### DISCUSSION TITLE
Backing camera freeze

### DISCUSSION THREAD

**Discussion by ukoda** [marcone/teslausb#848] [General]
I have sometimes had issues with my backing camera freezing up for a few seconds when backing into my garage.  I was wondering if anyone else had notice this?

I was wondering if it was a bug/limitation of the Tesla vehicle software with its handling of a USB drive being removed while the backing camera was active.  If so this could triggered by the teslausb software seeing my AP and 'unplugging' the USB flash drive before transferring files to my NAS.  If so it would not be a bug or issue with the teslausb software but could explain this long standing issue that Tesla have not fixed.  I would assume unplugging a flash drive while backing up a vehicles is probably not a use case Tesla tests for, hence it would explain why the problem still exists today.

Currently just a theory but was interested in if others had similar experiences?

**Comment by Arthur-Philip-Dent** [comment]
You drive a Model3? The Model3 had issues with the cable harness with the rear viewing camera. 
Could be this?

**Reply by ukoda** [reply]
Yes, a 2020 M3 Performance.  I did ask Tesla to look at just before the initial warranty expired.  The tech did a system reset and it was better for a while.  It is not as common as it was.  I thought I would save a recording to show them next time it happened and that is when I notice the USB drive was shown as unplugged.  Hence my wonder if it was related to it connecting the WiFI AP located above the garage.  I don't recall it ever freezing at any other location than my garage and freeze is only a few seconds.

**Comment by marcone** [comment]
> teslausb software seeing my AP and 'unplugging' the USB flash drive before transferring files to my NAS

When did you last install/update TeslaUSB?
TeslaUSB used to disconnect the drive before archiving, but for the past 2 years it's been doing it _after_ archiving.

**Reply by ukoda** [reply]
The software was updated fairly recently in trying to deal with the issue of missing files (now fixed).  I was making assumptions based on seeing the drive showing as unplugged after a camera freeze.  I figured I would post here to see if others had seen the same issue as a starting point.  I'm not expecting any changes to be made to TeslaUSB as I am assuming the root issue is with the vehicle.  Just looking for patterns or triggers given the intermittent nature of the problem.
